sphinx_simplepdf/themes/simplepdf_theme/__init__.py

```diff
@@ -14,10 +16,33 @@ def get_html_theme_path():
     return cur_dir
 
 
+def create_custom_css(app):
+
+    def get_config_var(name, default):
+        return app.config.simplepdf_vars.get(name, default)
+
+    def get_theme_var(name, default):
+        return app.config.simplepdf_theme_options.get(name, default)
+
+    here = path.abspath(path.dirname(__file__))
+    scss_folder = path.join(here, "static", "styles", "sources")
+
+    staticdir = path.join(app.builder.outdir, "_static")
+
+    sass.compile(
+        dirname=(scss_folder, staticdir),
+        output_style="nested",
+        custom_functions={
+            sass.SassFunction("config", ("$a", "$b"), get_config_var),
+            sass.SassFunction("theme_option", ("$a", "$b"), get_theme_var),
+        },
+    )
+
+
 # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
 def setup(app):
     app.add_html_theme('simplepdf_theme', path.abspath(path.dirname(__file__)))
-    # app.add_css_file('styles/main.css')
+    app.connect('builder-inited', create_custom_css)
```

Comment:
I think with this line we call `create_custom_css` every time, so for each builder including `html`.

We should add a check inside `create_custom_css` if the current builder is `simplepdf`.